fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1237,7 +1267,7 @@ public static class Builder {
             private DesiredAction ifReadable = DesiredAction.CONTINUE;
             private boolean doAllowUniquePendingState = false;
             private Set<TakeoverTypes> allowedTakeoverSet = null;
-            private long checkIndexingStampFrequency = 60_000;
+            private long checkIndexingStampFrequency = 10_000;
```

Comment:
Why check the type stamp more often?

Reply:
I had changed the default to check heartbeats and typestamp every 5 seconds. I would consider changing the default least time for longer than 10 seconds, but this should probably be done on a separate PR.

Reply:
For the record - now checking during every iterating transaction.